app/controllers/transaction_controller.rb

```diff
@@ -1,15 +1,58 @@
+
 class TransactionController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def index
+    @base = Transbank::Onepay::Base::DEFAULT_CALLBACK
+  end
 
   def create
+    Transbank::Onepay::Base.shared_secret = "?XW#WOLG##FBAGEAYSNQ5APD#JF@$AYZ"
+    Transbank::Onepay::Base.api_key = "dKVhq1WGt_XapIYirTXNyUKoWTDFfxaEV63-O5jcsdw"
+
+   # ENV["ONEPAY_API_KEY"] = "mUc0GxYGor6X8u-_oB3e-HWJulRG01WoC96-_tUA3Bg"
```

Comment:
Again, don't put commented out code in the repo unless you have a very good reason